tools/api-markdown-documenter/README.md

```diff
@@ -289,21 +289,49 @@ If you would like to add rendering support for a custom `Documentation Domain` n
 
 If you would like to change any or all of this library's default rendering policies, you may simply override the default policies for the desired `type`s.
 
-## HtmlRenderer
+### HtmlRenderer
 
-This library now includes APIs for HTML rendering.
+This library includes APIs for HTML rendering.
 
-Like the [MarkdownRenderer](#markdownrenderer), we offer a `HtmlRenderer.renderApiModel` function that operates in much the same way, but outputs `HTML`-formatted documents instead of Markdown.
+Like the [MarkdownRenderer](#markdownrenderer), this library includes a `HtmlRenderer.renderApiModel` function that operates in much the same way, but outputs HTML-formatted documents instead of Markdown.
+
+### ToHtml Transformation
```

Comment:
Note for reviewers: moved this out of the "Preview APIs" section - this API has been `@public` for a while.